### DISCUSSION TITLE
Github pro plan billing done incorrectly

### DISCUSSION THREAD

**Discussion by Matheussvini** [community/community#52187] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Hello everyone,

I recently purchased GitHub PRO on a monthly basis for $4, but when I received my credit card statement, I was charged for the annual plan at $48. I immediately noticed the mistake and opened a support ticket on May 1st, 2023, requesting a cancellation of the purchase. However, it's been 5 days now, and I have not received any response from the support team.

As a GitHub PRO customer, I expected to have access to exclusive and preferential support, but it seems like the support team is not fulfilling its promises. I am disappointed with this situation and would like to share my experience here to bring attention to the issue and hopefully get a resolution soon.

My ticket:
https://support.github.com/ticket/2088850

Thank you for your attention.

**Comment by queenofcorgis** [comment]
Hi there! Thanks for the feedback, it looks like your ticket was solved and I appreciate your patience as that got sorted. 

Just a quick reminder the Issues category is solely related to conversations around the GitHub products Issues and Projects.

**Reply by Matheussvini** [reply]
Hi Bro! Unfortunately no, my ticket still hasn't been resolved. I've been waiting for 12 days with no response from github support.
